src/stories/pages/guided-mode/GuidedHome.js

```diff
@@ -219,18 +219,15 @@ export class GuidedHomePage extends Page {
     return html`
         <div id="guided-home" class="guided--main-tab">
           <div class="guided--panel">
-            <h1 class="guided--text-sub-step">Guided Mode</h1>
-            <!-- <div class="title-border">
-              </div> -->
-            <p class="guided--help-text" style="margin-bottom: 2rem">
-              The Guided Mode is intended to guide users step-by-step through all the requirements for
-              curating and sharing datasets according to the SPARC data standards. The user interfaces
-              of the Guided Mode are designed to logically guide users through the curation steps and
-              include all necessary information such that no prior knowledge of the SPARC data
-              standards is required. Contrary to the Free Form Mode, the Guided Mode interfaces are
-              interconnected and form a single workflow such that the curation process is streamlined
-              further.
+          <h1 class="guided--text-sub-step">Guided Mode</h1>
+          <p class="guided--help-text" style="margin-bottom: 2rem">
+              NWB GUIDE is intended to guide users step-by-step through all the requirements for
+              converting and uploading datasets to the NWB format for the DANDI Archive. The user interfaces
+              of the Guided Mode are designed to logically guide users through the conversion process and
+              include all necessary information such that no prior knowledge of the NWB data
+              standard is required.
```

Comment:
```suggestion
              NWB GUIDE is intended to guide users step-by-step through all the requirements for
              converting and uploading datasets to the NWB format for the DANDI Archive. Each stage
              of the Guided Mode is designed to conveniently direct users through the conversion process and
              include all necessary information such that no prior knowledge of the NWB data
              standard is required.
```

What do you think of this rephrasing? Trying to reduce repetition of 'user' and 'guide'